### PR TITLE
Lint - increase attempts to 3 and log docker error in find py ver

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
increase number of attempts to find py ver to 3 and log the exception

### CODE REVIEW

**Review by bakatzir — APPROVED** [review]

File: demisto_sdk/commands/lint/helpers.py [hunk 1]
```diff
@@ -282,8 +282,8 @@ def get_python_version_from_image(image: str) -> float:
     """
     docker_client = docker.from_env()
     py_num = 2.7
```
Comment (yaakovi) [inline demisto_sdk/commands/lint/helpers.py:284]: since more and more integration and scripts are using python3, should we switch the fallback to python3?

**Review by yaakovi — APPROVED** [review]

File: demisto_sdk/commands/lint/helpers.py (hunk 1 above)
Comment (Itay4) [reply demisto_sdk/commands/lint/helpers.py:284]: since most are on 3.8, ill set it to that

**Comment by guyfreund** [comment]
@Itay4 As agreed with @hod-alpert we can merge this PR in the meantime, we are not sure it is solving the issue but it provides more logs and better error handling so in the next case of failure will know more.

File: demisto_sdk/commands/common/logger.py [hunk 2]
```diff
@@ -32,7 +32,8 @@ def logging_setup(verbose: int, quiet: Optional[bool] = False,
     # Setting debug log file if in circleci
     if log_path:
         file_handler = logging.FileHandler(filename=os.path.join(log_path, 'lint_debug_log.log'))
-        file_handler.setFormatter(fmt)
+        file_handler.setFormatter(
+            logging.Formatter('[%(asctime)s] - [%(threadName)s] - [%(levelname)s] - %(message)s'))
```
Comment (Itay4) [inline demisto_sdk/commands/common/logger.py:36]: @hod-alpert why not modify `fmt` ?

**Pull request merged**